packages/ui/src/utils/camel-uri-helper.ts

```diff
@@ -63,14 +63,22 @@ export class CamelUriHelper {
     }, {} as ParsedParameters);
   }
 
+  private static isExpression(value: string): boolean {
+    return /^\s*(\{\{.*\}\}|\$\{.*\})\s*$/.test(value);
```

Comment:
Since this code would run multiple times during typing, creating a new REGEXP would be expensive. What do you think about extracting this regexp into a module variable, or a class property, so we can reuse it? Ideally, after using it, we need to call `REGEXP.lastIndex=0;` to reset it.

As a friendly suggestion, these values are not necessarily suggestions but placeholders: `{{ }}`, maybe we can name the regexp `CONTAINS_PLACEHOLDER_REGEXP` or any other name you think is more suitable.